LICENSE.md

```diff
@@ -1,7 +1,7 @@
 The MIT License (MIT)
 =====================
 
-Copyright © 2016-2017 Michael Teeuw
+Copyright © 2016-2019 Michael Teeuw
```

Comment:
```suggestion
Copyright © Michael Teeuw
```

Reply:
Why?

Reply:
So no more year changes/updates..

Reply:
Ah, ok. But maybe this to help us to keep in mind about how long old is the project.

Reply:
Not so useful info.. if it makes sense.. add somewhere in the readme when the project started.. but you can always check the history..

Reply:
https://github.com/MichMich/MagicMirror/graphs/contributors

Reply:
Sound good, but I can see the project year in the copyright is more accord to http://www.gnu.org/licenses/gpl-howto.html